http_cache/esi.rst

```diff
@@ -234,7 +234,7 @@ that must be enabled in your configuration:
         <?xml version="1.0" encoding="UTF-8" ?>
         <container xmlns="http://symfony.com/schema/dic/services"
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:doctrine="http://symfony.com/schema/dic/framework"
+            xmlns:doctrine="http://symfony.com/schema/dic/doctrine"
```

Comment:
This change looks wrong to me. Instead, the namespace should be changed to `framework`:

```
xmlns:framework="http://symfony.com/schema/dic/framework"
```